src/commands/env-configs/list.ts

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import {FlagOutput} from '@oclif/core/lib/interfaces'
+import makeDebug from 'debug'
+
+import * as MixFlags from '../../utils/flags'
+import MixCommand from '../../utils/base/mix-command'
+import {MixClient, MixResponse, MixResult} from '../../mix/types'
+import {
+  EnvConfigListResponse,
+  EnvConfigListTransformedData,
+  EnvConfigsListParams,
+} from '../../mix/api/env-configs-types'
+import {listEnvConfigs} from '../../mix/api/env-configs'
+
+const debug = makeDebug('mix:commands:env-configs:list')
+
+export default class EnvConfigsList extends MixCommand {
+  static description = `list environment configurations
+  
+  Use this command to list all environment configurations for a specific project.`;
```

Comment:
Remove trainling semicolon;

Same for lines 32 and 39.

Rationale: the code base is semicolon-less.